crates/bevy_gilrs/src/gilrs_system.rs

```diff
@@ -6,7 +6,7 @@ use gilrs::{EventType, Gilrs};
 
 pub fn gilrs_event_startup_system(gilrs: NonSend<Gilrs>, mut events: EventWriter<GamepadEventRaw>) {
     for (id, _) in gilrs.gamepads() {
-        events.send(GamepadEventRaw(
+        events.send(GamepadEventRaw::new(
```

Comment:
On review, I think this is nicer than the tuple structs.

Reply:
Not sure if I like it better than a named struct though.

Reply:
In my eyes tuple structs should be avoided in most cases, because they are a pain to work with. I'm fine with both the named struct and the `new()` constructor. Would probably be a good idea to have a place where we note things like this so we can keep things consistent.

Reply:
Yeah, we have an engine style guide: https://github.com/bevyengine/bevy/blob/main/.github/contributing/engine_style_guide.md

@cart, do you have strong feelings?